src/vs/server/node/webClientServer.ts

```diff
@@ -353,9 +354,11 @@ export class WebClientServer {
 			'manifest-src \'self\';'
 		].join(' ');
 
+		const allowAllCSP = `default-src * 'unsafe-inline' 'unsafe-eval'; script-src * 'unsafe-inline' 'unsafe-eval'; connect-src * 'unsafe-inline'; img-src * data: blob: 'unsafe-inline'; frame-src *; style-src * 'unsafe-inline';`;
+
 		const headers: http.OutgoingHttpHeaders = {
 			'Content-Type': 'text/html',
-			'Content-Security-Policy': cspDirectives
+			'Content-Security-Policy': this._environmentService.isBuilt ? cspDirectives : allowAllCSP
```

Comment:
Add this to make workbench.html `onerror` works in dev mode. `allowAllSCP` value comes from [stackoverflow](https://stackoverflow.com/a/47598529)